vllm/multimodal/utils.py

```diff
@@ -0,0 +1,71 @@
+import base64
+from io import BytesIO
+from typing import Union
+
+import requests
+from PIL import Image
```

Comment:
can you double check this is in requirements txt

Reply:
This was added in the last image processor PR. https://github.com/vllm-project/vllm/blob/7b0a0dfb22907505441f8a4a5eb882cbca4d2acf/requirements-common.txt#L15